sdk/core/core-amqp/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 - Changed TS compilation target to ES2017 in order to produce smaller bundles and use more native platform features
 - With the dropping of support for Node.js versions that are no longer in LTS, the dependency on `@types/node` has been updated to version 12. Read our [support policy](https://github.com/Azure/azure-sdk-for-js/blob/main/SUPPORT.md) for more details.
 - Updated to use the latest version of the `rhea` package.
+- Add a constant `messageState` with value of `"x-opt-message-state"`.
```

Comment:
nit

Add the link to the PR too

Reply:
yeah I need to have an entry in SB CHANGELOG too